src/test/java/bio/terra/service/rawls/RawlsClientTest.java

```diff
@@ -0,0 +1,87 @@
+package bio.terra.service.rawls;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import bio.terra.app.configuration.RawlsConfiguration;
+import bio.terra.common.HttpEntityUtils;
+import bio.terra.common.category.Unit;
+import bio.terra.common.fixtures.AuthenticationFixtures;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class RawlsClientTest {
+
+  @Mock private RawlsConfiguration rawlsConfiguration;
+  @Mock private RestTemplate restTemplate;
+  private RawlsClient rawlsClient;
+
+  private static final String BASE_PATH = "rawls.base.path";
+  private static final AuthenticatedUserRequest TEST_USER =
+      AuthenticationFixtures.randomUserRequest();
+  private static final UUID WORKSPACE_ID = UUID.randomUUID();
+  private static final String NAMESPACE = "namespace";
+  private static final String NAME = "name";
+
+  @BeforeEach
+  void beforeEach() {
+    when(rawlsConfiguration.basePath()).thenReturn(BASE_PATH);
```

Comment:
It works either way but in general I wouldn't mock a POJO/record when you can create it directly instead.
```suggestion
    var rawlsConfiguration = new RawlsConfiguration(BASE_PATH);
```